src/content/data-table-utils.js

```diff
@@ -64,7 +65,7 @@ export function sortDataTable<KeyColumnElementType>(
 
   function quickSort(left, right) {
     if (left < right) {
-      const pivot = right;
+      const pivot = (left + right) >> 1;
```

Comment:
Thanks Jim, that's really a great comment! I'm putting it into the code verbatim.